test_ir.py

```diff
@@ -37,5 +37,68 @@ class DisplaySystem(jkt.KernelTests):
     {'code': '1:3', 'mime': {'text/plain':'[1] 1 2 3'}},
     ]
 
+class AbstractIRKernel(jkt.KernelTests):
```

Comment:
Doing it like this with a separate class for each test means that it re-discovers and then skips the base jupyter_kernel_test tests for each subclass. Is that intentional?